main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -146,7 +146,7 @@ object Main {
               val result = bootstrap.check(options)
               System.exit(getCode(result))
             case Result.Err(e) =>
-              println(e)
+              println(e.message(Formatter.NoFormatter))
```

Comment:
I think, somewhere above the try, it would nice to have:

```
    if (Formatter.hasColorSupport)
      flix.setFormatter(AnsiTerminalFormatter)
```

Reply:
Oops. instead of `flix.setFormatter`, you would have something like:

```
val formatter = if (...) AnsiTerminalFormatter else NoFormatter

Reply:
I will merge now, and then we can follow up on this.